assets/component-facets.css

```diff
@@ -903,3 +932,164 @@ input.mobile-facets__checkbox {
 .product-count__text.loading + .loading-overlay__spinner {
   display: block;
 }
+
+@media screen and (min-width: 750px) {
+  .facets-vertical {
+    display: flex;
+  }
+
+  .facets-vertical .disclosure-has-popup[open] > summary::before {
+    content: none;
+  }
+  
+  .facets-vertical .facets__display {
+    position: initial;
+    border: none;
+    max-height: none;
+    width: 26rem;
+  }
+    
+  .facets-vertical .facets__form {
+    display: flex;
+    flex-direction: column;
+    width: 26rem;
+  }
+  
+  .facets-vertical .facets__wrapper {
+    flex-direction: column;
+    align-items: initial;
+    align-self: initial;
+    flex-wrap: initial;
+  }
+  
+  .facets-vertical .facets__disclosure {
+    border-top: solid rgba(var(--color-foreground),var(--popup-border-opacity));
```

Comment:
Thoughts on using the same borders as the Collapsible content section's accordion?

- https://github.com/Shopify/dawn/blob/3e5c4f8fb5cdef5db1cc6f4bb1a5eb081c5c29f7/assets/component-accordion.css#L22-L27

I was testing on Firefox and Safari and noticed the borders looked "thicker":

<details>
  <summary>Firefox</summary>

  ![image](https://user-images.githubusercontent.com/28404165/159562058-4574988e-831f-4618-a597-58159700ef1f.png)

</details>

<details>
  <summary>Safari</summary>

  ![image](https://user-images.githubusercontent.com/28404165/159562118-e4326a49-202f-453d-9112-94925f559cb6.png)

</details>

I noticed something re: the first filter being open (we chatted about this earlier) but I'll add that as a comment on your PR 👍🏼

Reply:
Hey @duygukalaycioglu - I don't see this as a blocker myself, how about you? I can see this as something we can quickly fix in a follow up issue.